upgrade/2022/r3-2022.md

```diff
@@ -0,0 +1,133 @@
+---
+title: R3 2022
+page_title: R3 2022 
+description: R3 2022
+slug: telerikreporting/upgrade/2022/r3-2022
+tags: r3,2022
+published: True
+position: 4
+---
+
+# R3 2022
+
+This article explains the manual changes required when upgrading to Telerik Reporting R3 2022 (16.2.22.914).
+
+## Changes
+
+### Native Blazor Report Viewer
+
+The viewer is built with Telerik UI for Blazor __3.5.0__.
+
+### WinUI Report Viewer
+
+The viewer is built with Telerik UI for WinUI __2.2.0__. It now uses __MicrosoftWindowsAppSDK 1.1.0__.
+
+### WPF Report Viewer
+
+### WPF Report Viewer for .NET Framework
+
+The viewer is built with Telerik UI Controls for WPF __2022.3.913.40__. If you are using a newer version, consider adding binding redirects. For more information see: [How to Add report viewer to a WPF .NET Framework project]({%slug telerikreporting/using-reports-in-applications/display-reports-in-applications/wpf-application/how-to-add-report-viewer-to-a-wpf-.net-framework-project%}).
```

Comment:
I would say "If you use ..."

Reply:
I just copied the text from our previous UpgradePath article. I think the present continuous is more correct here, because it reflects an event that is currently happening, whereas present simple is used when the subject does not change. Not really sure, though, will ask a native speaker for that (Rick, probably). Thanks for mentioning it!